src/Clerk.ts

```diff
@@ -187,18 +187,15 @@ export default class Clerk {
     return decoded as JwtPayload;
   }
 
-  async verifyToken(
-    token: string,
-    algorithms: string[] = ['RS256']
-  ): Promise<JwtPayload> {
+  async verifyToken(token: string): Promise<JwtPayload> {
     const decoded = jwt.decode(token, { complete: true });
     if (!decoded) {
       throw new Error(`Failed to verify token: ${token}`);
     }
 
     const key = await this._jwksClient.getSigningKey(decoded.header.kid);
     const verified = jwt.verify(token, key.getPublicKey(), {
-      algorithms: algorithms as jwt.Algorithm[],
+      algorithms: ['RS256'],
```

Comment:
❓ Are we planning to use this method for other tokens except session token? If yes I think we should keep the option. cc: @chanioxaris, @igneel64

Reply:
I don't think that this function should be generic and validating other tokens than the clerk session tokens. It already contains specific validation rules (e.g. issuer). If someone wants to verify any token, can use a popular node package such as `jsonwebtoken`, which is the one we use internally

Reply:
Great. Let's remove it then.

Reply:
This is the public method which is exposed from the package so I am not sure if there is a change we will be using this for another token.